fish.py

```diff
@@ -456,8 +477,10 @@ def blowcrypt_unpack(msg, cipher):
     except ValueError:
         raise MalformedError
 
-    return plain.strip('\x00').replace('\n',''), broken
+    if wrong_key_type:
```

Comment:
minor nitpick here, this if+return is redundant as it is identical to the other return in line 483